sdk/ml/azure-ai-ml/azure/ai/ml/operations/_operation_orchestrator.py

```diff
@@ -356,6 +372,9 @@ def _resolve_name_version_from_name_label(self, aml_id: str, azureml_type: str)
         ):
             return aml_id, None
 
+        if label == "default":
```

Comment:
Will this line affect other type of assets which also has default label, like Data?

Reply:
sorry, this line belongs to an earlier implementation and should be removed.